docs/development/dev_box_de.md

```diff
@@ -49,4 +50,18 @@ Zum Entschlüssen kann folgender Befehl verwendet werden:
 gpg --decrypt .vagrant.rb.asc > .vagrant.rb
 ```
 
-> **Hinweis:** Bei Änderungen in der `.vagrant.rb` muss diese erneut verschlüsselt und anschließend gelöscht werden! 
\ No newline at end of file
+> **Hinweis:** Bei Änderungen in der `.vagrant.rb` muss diese erneut verschlüsselt und anschließend gelöscht werden!
+
+### Zertifikate
+Standardmäßig erstellt das Setup des CES ein selbstsigniertes SSL-Zertifikat für die Absicherung der HTTPS-Verbindungen.
```

Comment:
Does this only happen if the user does not provide a valid certificate? Or what happens in the case of production? Perhaps you should make this sentence a little clearer so that the situation is unambiguous.